spec/models/draw_spec.rb

```diff
@@ -16,4 +17,47 @@
       expect(draw.suite_sizes).to match_array([1, 2, 3])
     end
   end
+
+  # TODO: should we keep this? I originally wrote this before I decided to make
+  # #student_count private, but we can make it public again if we think it's
+  # worth it.
+  #
```

Comment:
might as well keep it, it doesn't persist data so it's fast, and we can always delete it later